app/services/declarations/create.rb

```diff
@@ -28,7 +30,7 @@ class Create
     validate :output_fee_statement_available
     validate :validate_has_passed_field, if: :validate_has_passed?
     validate :validates_billable_slot_available
-    validate :declaration_date_not_in_the_future
```

Comment:
this validation is in the declaration validations, which are now checked in this service